src/main/java/de/tum/in/www1/artemis/web/rest/StudentExamResource.java

```diff
@@ -890,4 +890,17 @@ public ResponseEntity<StudentExam> unsubmitStudentExam(@PathVariable Long course
 
         return ResponseEntity.ok(studentExamRepository.save(studentExam));
     }
+
+    /**
+     * GET exams/{examId}/longest-working-time : Returns the value of
+     * the longest working time of the exam
+     *
+     * @param examId the exam to which the student exams belong to
+     * @return the longest working time of the exam (in seconds)
+     */
+    @GetMapping("/exams/{examId}/longest-working-time")
+    public ResponseEntity<Integer> getLongestWorkingTimeForExam(@PathVariable Long examId) {
```

Comment:
You need authentication for this endpoint

Reply:
Added